src/ga/ga_simple.rs

```diff
@@ -101,7 +101,9 @@ impl<T: GAIndividual + Clone> GeneticAlgorithm<T> for SimpleGeneticAlgorithm <T>
         let best_old_individual = self.population.best().clone();
 
         // Evaluate the new population
-        self.population.swap(new_individuals);
+        // TODO: Archive the old population
+        let order = self.population.order();
+        self.population = GAPopulation::new(new_individuals, order);
```

Comment:
@carlos-lopez-garces I think you are right about what you said in the previous commit (with regards to just using a new GAPopulation instead of risking a de-sync between statistics and the population). This should work with what you planned, right?